bitcointranscripts/transcription-review-front-end
# Guidelines modal

How to test for first timers
1. Make sure you don't have any reviews either pending, inactive or active, if you have time you could negate the code in queuetable.tsx line 194 `  if (multipleStatusData.length > 0) {`

How to test for old timers
1. There is an orange review Guidlines button at the top of the editor, click that and you have your modal


we could change the design going forward.

TESTED.
